spec/mailers/user_mailer_spec.rb

```diff
@@ -762,6 +762,53 @@ def expect_email_body_to_have_help_and_contact_links
         end
       end
 
+      context 'Need to change location section' do
+        context 'when Enhanced IPP is not enabled' do
+          let(:is_enhanced_ipp) { false }
+          before do
+            @is_enhanced_ipp = false
+          end
```

Comment:
Should this be using the assigned `let` variable assignment, or at the very least have a matching `before` in "when enabled" block below? Assigning with the variable would let us define it once and have the context variable change for each spec.

```suggestion
        before do
          @is_enhanced_ipp = is_enhanced_ipp
        end

        context 'when Enhanced IPP is not enabled' do
          let(:is_enhanced_ipp) { false }
```

Reply:
Good catch! These new tests were actually not functioning properly at all, thanks for highlighting them.